email/ally-fulfilment-inline.html

```diff
@@ -0,0 +1,236 @@
+                                                    </td>
+                                                </tr>
+                                                <tr>
+                                                    <td style="border-collapse:collapse;font-family:'Source Sans Pro', sans-serif;font-size:16px;vertical-align:top;">
+                                                        <a class="url" href="#" title="http://institition.edu" target="_blank" style="color:#191F25;color:#FFFFFF;display:block;font-size:17px;font-weight:bold;text-decoration:underline;padding-bottom:40px;">http://institition.edu</a>
+                                                    </td>
+                                                </tr>
+                                                <tr>
+                                                    <td class="hero-btn" style="border-collapse:collapse;font-family:'Source Sans Pro', sans-serif;font-size:16px;vertical-align:top;">
+                                                        <table border="0" cellpadding="0" cellspacing="0" class="btn btn-primary" style="border-collapse:collapse;mso-table-lspace:0pt;mso-table-rspace:0pt;box-sizing:border-box;width:100%;text-align:center;padding:16px;Margin-bottom:40px;">
+                                                            <tbody>
+                                                                <tr>
+                                                                    <td style="border-collapse:collapse;font-family:'Source Sans Pro', sans-serif;font-size:16px;vertical-align:top;">
+                                                                        <a href="http://usergroup.ally.ac" title="Request an Ally demonstration" target="_blank" style="color:#191F25;text-decoration:underline;background-color:#FFFFFF;box-sizing:border-box;color:#007900;cursor:pointer;display:inline-block;font-size:15px;font-weight:bold;line-height:1.6;margin:0;padding:8px 16px;text-decoration:none;text-transform:capitalize;background-color:#44474F;border-color:#44474F;color:#FFFFFF;background-color:#007900;border:solid 1px #007900;">View Ally documentation</a>
+                                                                    </td>
+                                                                </tr>
+                                                            </tbody>
+                                                        </table>
+                                                    </td>
+                                                </tr>
+                                                <tr>
+                                                    <td style="border-collapse:collapse;font-family:'Source Sans Pro', sans-serif;font-size:16px;vertical-align:top;">
+                                                    </td>
```

Comment:
These empty cells could be an accessibility issue, but I have no idea at what point that actually matters in a table layout for email. A more semantic approach if we're just looking for empty space would be to use margin/padding.